ckanext/resourceproxy/controller.py

```diff
@@ -58,7 +58,7 @@ def proxy_resource(context, data_dict):
         if not did_get:
             r = requests.get(url, stream=True)
 
-        base.response.content_type = r.headers['content-type']
+        base.response.content_type = r.headers.get('content-type', '')
```

Comment:
I did some test and it looks like my version of [Requests](http://docs.python-requests.org/en/master/) is out of date. By updating it I fixed the problem.

Sorry about that. I guess there is no need to fix then.